mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -680,7 +684,7 @@ def images_to_video(input_folder: str,
             Defaults to 0.
         end (int, optional): end frame index. Included.
             If < 0, will be converted to frame_index range in [0, frame_num].
-            Defaults to -1.
+            Defaults to None.
```

Comment:
The docstring for `end` is different for different functions, may consider use the same description for consistency.

Reply:
solved